prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -280,7 +280,7 @@ ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	if (inject)
 		err = rxr_pkt_entry_inject(rxr_ep, pkt_entry, addr);
```

Comment:
Adding flags to the function definition but not the inject path is a bit misleading and could hide a bug in the future. Maybe change `rxr_pkt_entry_inject` to use send with the right inject flags instead of calling `fi_inject()`?

Reply:
Based on offline discussions, I added an `assert(!flags)` in the inject path to make sure `FI_MORE` is not added to flags in this case.

Reply:
I am not sure the comment is correct. I think the reason flags must be 0 here is because this function currently only take FI_MORE as flag, and FI_MORE is not compatible with inject. It has nothting to do with shm and EOR

Reply:
Fixed the comment in the latest revision .